internal/utils/dbidentifier/dbidentifier_test.go

```diff
@@ -22,4 +23,7 @@ func TestToGoIdentifier(t *testing.T) {
 	require.Equal(t, ToGoIdentifier("My_Table"), "MyTable")
 	require.Equal(t, ToGoIdentifier("My Table"), "MyTable")
 	require.Equal(t, ToGoIdentifier("My-Table"), "MyTable")
+
+	require.Equal(t, ToGoIdentifier("My-Table+"), "MyTablePlus")
+	require.Equal(t, ToGoIdentifier("My-Table-"), "MyTableMinus")
```

Comment:
It feels like quick fix for your use case. But there are many more. The invalid character can also appear at the start or at the middle of identifier. From the perspective of this library, jet will replace all go identifier invalid characters with `_`(There is an improvement for this at branch [bug290](https://github.com/go-jet/jet/commit/48a3e78dc714414e8910e1c57f7febab1364917c), yet to be merged to master).   
And then if there is a conflict developer can use [generator customization](https://github.com/go-jet/jet/wiki/Generator#generator-customization) to replace conflicting name with a name of his choosing.

Reply:
Makes sense, I'll workaround using generator customization for now.